app/views/layouts/application.html.erb

```diff
@@ -223,5 +223,6 @@
     <%= yield :javascript %>
     <script type="text/javascript" async defer id="gauges-tracker" data-site-id="4eab0ac8613f5d1583000005" src="//secure.gaug.es/track.js"></script>
     <script type="text/javascript" defer src="https://www.fastly-insights.com/insights.js?k=3e63c3cd-fc37-4b19-80b9-65ce64af060a"></script>
+    <script type="text/javascript" src="https://unpkg.com/@github/webauthn-json@0.3.1/dist/index.js"></script>
```

Comment:
Is this OK or should be use `vendor/assets/javascripts`?

Reply:
This grants anyone with deploy access at `unpkg.com` the ability to steal credentials from every account.

Reply:
Good call. Needs to change. Thank you :+1:

Reply:
For that matter, the `secure.gaug.es` stuff stands out to me as pretty high-risk also. I'm sure the staff there are all very fine people, but they still shouldn't have access to my unhashed password.